compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.uikit.kt

```diff
@@ -345,7 +336,7 @@ internal class ComposeSceneMediator(
     val hasInvalidations: Boolean
         get() = scene.hasInvalidations() || keyboardManager.isAnimating
 
-    private fun hitTestInteropView(point: CValue<CGPoint>, event: UIEvent?): UIView? =
```

Comment:
Please revert unrelated changes

Reply:
That was an attempt to fix warnings in the file. Reverted.